cpp/include/cudf/io/orc.hpp

```diff
@@ -709,6 +859,42 @@ class chunked_orc_writer_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Sets the maximum stripe size, in bytes.
+   *
+   * @param val maximum stripe size
+   * @return this for chaining.
+   */
+  chunked_orc_writer_options_builder& enable_statistics(size_t val)
+  {
+    options.set_stripe_size_bytes(val);
+    return *this;
+  }
+
+  /**
+   * @brief Sets the maximum number of rows in output stripes.
+   *
+   * @param val maximum number or rows
+   * @return this for chaining.
+   */
+  chunked_orc_writer_options_builder& enable_statistics(size_type val)
```

Comment:
shouldn't these functions be renamed to 
```
stripe_size_bytes(size_t val) 
stripe_size_rows(size_type val)
```
?
Also, are these functions tested in unit tests?

Reply:
Yup, that's a copy-paste error.
I haven't added tests for the chunked writer, since the code path in the writer is the same. But it looks like there's still room for error :(
I'll see if if can add a meaningful test for the chunked writer, to exercise these functions as well.